Dockerfile

```
@@ -0,0 +1,30 @@
+FROM node:18.20-alpine AS build-stage
+
+RUN apk update  \
+  && apk add ruby
+
+COPY pix-site code/pix-site
+COPY shared code/shared
+
+ENV SITE=pix-site
+
+WORKDIR /code/shared
+RUN npm install
+
+WORKDIR ../pix-site
+RUN npm install
+RUN npm run build
```

Il faut plutôt utiliser `npm ci` qui lui utilisera les versions lockées des packages. 

```suggestion
ENV SITE=pix-site

WORKDIR /code/shared
RUN npm ci

WORKDIR ../pix-site
RUN npm ci
RUN npm run build
```